tests/Integration/Execution/MutationExecutor/HasManyTest.php

```diff
@@ -116,6 +121,53 @@ public function testCanCreateWithNewHasMany(): void
         ]);
     }
 
+    public function testCanCreateWithConnectHasMany(): void
+    {
+        $task1 = factory(Task::class)->create();
+        $task2 = factory(Task::class)->create();
+        $task3 = factory(Task::class)->create();
+
+        $this->graphQL(/** @lang GraphQL */ <<<GRAPHQL
+        mutation {
+            createUser(input: {
+                name: "foo"
+                tasks: {
+                    connect: [ $task1->id, $task2->id, $task3->id ]
```

Comment:
Please use actual query variables instead of string interpolation.

Reply:
Done, I have modified it.